src/autoscaler/eventgenerator/generator/evaluator_test.go

```diff
@@ -613,6 +607,7 @@ var _ = Describe("Evaluator", func() {
 							errorStr = errorStr + tmp
 						}
 						scalingEngine.RouteToHandler("POST", urlPath, ghttp.RespondWithJSONEncoded(http.StatusBadRequest, errorStr))
+						Expect(triggerChan).To(BeSent(triggerArrayGT))
```

Comment:
looks like we're fixing a pended test here.. does anyone know/remember why it's pended? if we never run it it doesn't seem like it gives us much useful regression coverage, maybe we should either fix it or delete it? I tried running it locally and it seems to consistently fail..